dags/common/rest_api_service.py

```diff
@@ -0,0 +1,35 @@
+class RestApi:
+    def __init__(
+        self,
+        base_url: str = 'localhost:8080',
+        parameters: dict = {},
+        headers: dict = {}
+    ):
+        self.base_url = base_url 
+        self.parameters = parameters
+        self.headers = headers
+
+    def __repr__(self) -> str:
+        return(f"< RestApi base_url={self.base_url}, parameters={self.parameters}, header={self.headers} >")
+        
+    def get_response(self):
+        from furl import furl
+        import requests
```

Comment:
why importing inside the function?

Reply:
https://airflow.apache.org/docs/apache-airflow/stable/best-practices.html#top-level-python-code

Reply:
good to know thanks 👍